data-collection/src/main/java/refactoringml/PMDatabase.java

```diff
@@ -6,7 +6,12 @@
 import java.util.stream.Collectors;
 
 public class PMDatabase {
-
+	/*
+	Maps file names onto their original process metrics.
+	TODO: Track both class and filename renames
+	 I found that we currently only use filenames instead of classnames to track process metrics in the `PMDatabase.java`.
```

Comment:
I think we have no other option. Process metrics of the file == process metrics of the class. Indeed, if a file has more than one class, both classes have the same process metrics.

But you raise a good point: if there's a Rename Class, are we tracking it correctly?

Reply:
So, kinda test all the combinations, right?

File renamed, class not renamed
File not renamed, class renamed
File renamed, class renamed
(and file not renamed, class not renamed)

Reply:
I wrote tests for all those cases and the tracking works, but I am not entirely sure about the Process Metrics calculation, but I can have a look at that next week